core_contracts/daofund/daofund.py

```diff
@@ -77,6 +81,7 @@ def __init__(self, db: IconScoreDatabase) -> None:
         self._admin = VarDB(self._ADMIN, db, value_type=Address)
         self._loans_score = VarDB(self._LOANS_SCORE, db, value_type=Address)
         self._fund = DictDB(self._FUND, db, value_type=int)
+        self._symbol = ArrayDB(self._SYMBOL, db, value_type=str)
```

Comment:
Let's make this an enumerable SetDB since the ArrayDB does not scale. If you have time for that. We can try and see if it will be accepted, but it will probably need an update at some point anyway.
https://github.com/balancednetwork/balanced-contracts/blob/development/core_contracts/dex/scorelib/enumerable_set.py

Reply:
enumerable SetDB added. Let me know if I am missing something.